packages/vue/src/components/molecules/SfRadio/SfRadio.stories.js

```diff
@@ -90,16 +97,17 @@ const Template = (args, { argTypes }) => ({
   props: Object.keys(argTypes),
   template: `
   <SfRadio 
-  :class="classes"
-  :label="label"
-  :details="details"
-  :description="description"
-  :name="name"
-  :value="value"
-  :disabled="disabled"
-  :required="required"
-  v-model="selected"
-  @input="input"
+    :class="classes"
+    :label="label"
+    :details="details"
+    :description="description"
+    :name="name"
+    :value="value"
+    :disabled="disabled"
+    :required="required"
+    v-model="selected"
+    @change="change"
```

Comment:
In the future please make sure not to mix changes added to the code with linting/prettiering. I could probably miss the @change if it wasn't mentioned in the Pull Request 😉